### PR TITLE
创建issue反馈模板

### PR DESCRIPTION
**Pull request by RainyHallways**
创建issue反馈模板,可在[此处](https://github.com/RainyHallways/ifc/issues/new?assignees=&labels=&template=def.yml)查看效果

### CODE REVIEW

**Comment by RainyHallways** [comment]
附:能够自动添加标签

File: .github/ISSUE_TEMPLATE/def.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: "综合反馈"
+description: "提出疑问或反馈Bug"
+body:
+- type: checkboxes
+  id: "yml-1"
+  attributes:
+    label: "检查项"
+    description: "请检查下列项目，并勾选确认。"
+    options:
+    - label: "我已在 [Issues 页面](https://github.com/LNSSPsd/PhoenixBuilder/issues?q=is%3Aissue+) 中搜索，确认了这一 Bug 未被提交过。"
```
Comment (Torrekie) [inline .github/ISSUE_TEMPLATE/def.yml:10]:
```suggestion
    - label: "我已在 [Issues 页面](https://github.com/LNSSPsd/PhoenixBuilder/issues?q=is%3Aissue+) 中搜索，确认了在已激活的议题中这一 Bug 未被提及过。"
```

File: .github/ISSUE_TEMPLATE/def.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  attributes:
+    label: 描述
+    description: "请详细描述您想要反馈的项目"
+  validations:
+    required: true
+- type: textarea
+  id: "yml-3"
+  attributes:
+    label: 附件
+    description: "上传其余附件"
```
Comment (Torrekie) [inline .github/ISSUE_TEMPLATE/def.yml:23]:
```suggestion
    description: "上传其余附件（运行日志等）"
```

**Review by Torrekie — CHANGES_REQUESTED** [review]

**Review by Torrekie — APPROVED** [review]

**Comment by LNSSPsd** [comment]
Issues are human-reviewed, and maybe it's better to give more freedom in format.

**Pull request closed without merging**